### PR TITLE
Add workflow to delete old docker images

### PR DESCRIPTION
**Pull request by sylwiaszunejko**

### CODE REVIEW

File: .github/workflows/clean_dockerhub_images.yml
Comment (sylwiaszunejko) [inline .github/workflows/clean_dockerhub_images.yml:8]: only for the testing purpose, remove before merging

**Comment by sylwiaszunejko** [comment]
it works, it run successfully removed one of the images https://github.com/scylladb/gocql/actions/runs/10939117200/job/30368694202

File: .github/workflows/clean_dockerhub_images.yml
Comment (dkropachev) [reply .github/workflows/clean_dockerhub_images.yml:8]: please remove it before merging.

**Review by dkropachev — APPROVED** [review]

**Comment by sylwiaszunejko** [comment]
@dkropachev what should I put in `DELETE_AFTER_DAYS`? Now it is set to 4 to have something to remove, but it is probably to small, would 30 days be good?

**Comment by dkropachev** [comment]
> @dkropachev what should I put in `DELETE_AFTER_DAYS`? Now it is set to 4 to have something to remove, but it is probably to small, would 30 days be good?

Ah yes, let's set it to 30 days

**Comment by sylwiaszunejko** [comment]
@dkropachev It need to be approved again after the push

**Review by dkropachev — APPROVED** [review]

**Pull request merged**
